assets/web-ui/index.html

```diff
@@ -62,6 +62,13 @@ <h1>SPX Control Panel</h1>
                                 defaultValue: '0',
                                 description: 'Whether to enable SPX profiler for your current browser session. No performance impact for other clients.',
                             },
+                            {
```

Comment:
This should probably default to 1 rather than 0, as that will be more useful to the majority of people using  the profiler.

Reply:
Good catch :+1: